qemu.go

```diff
@@ -106,6 +108,12 @@ const (
 	maxDevIDSize = 31
 )
 
+const (
+	// NVDIMM device needs memory space (1024MB is suggested by @grahamwhaley)
+	// See https://github.com/clearcontainers/runtime/issues/380#issuecomment-321216520
+	defaultNvdimmMemory = 1024
```

Comment:
I would call this constant `maxMemoryOffset` because that's basically what it is, and we don't need to specify the reason in the variable name. As long as we have your comment, a generic name sounds better to me.